app/assets/stylesheets/users.scss

```diff
@@ -60,7 +60,7 @@
 }
 
 .right-shift {
-  margin-left: 30px;
+  margin-left: 0;
```

Comment:
also, make checkbox inline with `remember me` heading
like: 
<img width="289" alt="Screenshot 2022-08-04 at 10 34 24 PM" src="https://user-images.githubusercontent.com/76155456/182909421-6a722fc1-f337-4165-9ff0-fd0c7512462c.png">

Reply:
Yes,
@Sarthaknegigit you need to change [here](https://github.com/CircuitVerse/CircuitVerse/blob/638e0d7e029f0fd4dd59f5fe4b57855e648636ee/app/views/users/sessions/new.html.erb#L39).
- Swap the `label` and `h6` position.
- `primary-checkpoint` class not be changed as it is being used in many other files which are aligned.
- You can make a new class by adding additional code for it.

If you got any other idea, try that as well!

Reply:
Sure, I am on it.

Reply:
@vedant-jain03 Can you suggest some name for the new class or some convention that I need to follow while naming the class?

Reply:
`session-primary-checkbox`

Reply:
@vedant-jain03 , Sure